.github/workflows/test.yml

```diff
@@ -53,36 +53,61 @@ jobs:
           macos-13,
           windows-latest,
         ]
+        wasmer-feature: [
+          wasmer_sys,
+          wasmer_wamr
+        ]
+        exclude:
+          # Our intended target for wasmer_wamr feature is iOS and Android.
+          # For now, wasmer_wamr tests are only run on macos-latest.
```

Comment:
Makes sense. I wonder if this could be done with an `include` rather than an `exclude` to extend the matrix? Would be a little more compact and easier to see what's happening

Reply:
switched to include